.github/ISSUE_TEMPLATE/enhancement.yml

```diff
@@ -1,19 +1,14 @@
-name: "🚀 Enchantment"
+name: "🚀 Enhancement"
 description: Suggest an enchantment that will improve NuxtAuth
-labels: ["pending", "enchantment"]
+labels: ["pending", "enhancement"]
 body:
   - type: markdown
     attributes:
       value: |
         Thank you for taking the time to fill out this feature request!
 
-        Please note: Only post enchanments that you have properly considered. This means:
-        - You have properly determined what problem your enchanment fixes
-        - How you would recommend implementing your enchancement
-        - How this change would impact NuxtAuth
-
-        If you are still unsure about your implementation or would like to discuss it prior, open a discussion first:
-        👉 https://github.com/sidebase/nuxt-auth/discussions/new?category=feature-requests
+        Please carefully read the Nuxt contribution docs before suggesting a new feature
+        👉 https://nuxt.com/docs/community/contribution/#creating-an-issue
```

Comment:
Why did you change the bullet-points? For me they seemed very good

Reply:
Because we removed the discussion section altogether. Therefore the link would no longer be functional.

Reply:
It's not about the discussions, it's about the text above with "Please note:"

Reply:
Ahh, okay! I did not understand this from the line you added the message! I re-added that part again 🥰